test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/EmailAddressGeneratorTest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.services.test.utils.core.random;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static uk.gov.justice.services.test.utils.core.random.RandomGenerator.EMAIL_ADDRESS;
+
+import java.lang.reflect.Constructor;
+
+import org.junit.Test;
+
+public class EmailAddressGeneratorTest {
+
+    @Test
+    public void shouldCheckConstructorIsNotAccessible()
+                    throws NoSuchMethodException, SecurityException {
+        Constructor<EmailAddressGenerator> c = EmailAddressGenerator.class.getDeclaredConstructor();
+        assertFalse(c.isAccessible());
+    }
+
+    @Test
```

Comment:
yeah I agree...the test would still work if all the tests just contain "@" in them. Better to come up with a regex to validate against.